core/src/test/java/org/elasticsearch/search/aggregations/metrics/tophits/InternalTopHitsTests.java

```diff
@@ -54,11 +57,19 @@
      * Should the test instances look like they are sorted by some fields (true) or sorted by score (false). Set here because these need
      * to be the same across the entirety of {@link #testReduceRandom()}.
      */
-    private final boolean testInstancesLookSortedByField = randomBoolean();
+    private boolean testInstancesLookSortedByField;
     /**
      * Fields shared by all instances created by {@link #createTestInstance(String, List, Map)}.
      */
-    private final SortField[] testInstancesSortFields = testInstancesLookSortedByField ? randomSortFields() : new SortField[0];
+    private SortField[] testInstancesSortFields;
+
+    @Override
+    @Before
```

Comment:
Luca mentioned in another PR that we might not need the need @Bfore here since  the parent method already has it.